console/services/market_app/component.py

```diff
@@ -113,6 +115,7 @@ def _update_deploy_version(self, dv):
         if not dv["is_change"]:
             return
         self.component.deploy_version = dv["new"]
+        self.action_type = ActionType.BUILD.value
```

Comment:
This value is overwritten when other changes occur at the same time.

Reply:
fixed